src/tests/SimTest.cpp

```diff
@@ -160,9 +160,7 @@ void SimTest::reset() {
   Simulator simulator(cfg);
   PathFinder::ptr pathfinder = simulator.getPathFinder();
 
-  auto pinholeCameraSpec = SensorSpec::create();
-  pinholeCameraSpec->sensorSubType = esp::sensor::SensorSubType::Pinhole;
-  pinholeCameraSpec->sensorType = SensorType::Color;
```

Comment:
I understand you have default values. 
But still keep them here because it increases the readability. 
The same everywhere.